main.go

```diff
@@ -57,6 +57,12 @@ var (
 	serverAllowedDNSs    = serverCommand.Flag("allow-dns-san", "Allow clients with given DNS subject alternative name (can be repeated).").PlaceHolder("SAN").Strings()
 	serverAllowedIPs     = serverCommand.Flag("allow-ip-san", "Allow clients with given IP subject alternative name (can be repeated).").PlaceHolder("SAN").IPList()
 
+	clientCommand       = app.Command("client", "Client mode (TCP listener -> TLS port).")
+	clientListenAddress = clientCommand.Flag("listen", "Address and port to listen on (HOST:PORT), or unix:PATH.").PlaceHolder("ADDR").Required().String()
+	// Note: can't use TCP for clientForwardAddress because we need to set the original string in tls.Config.ServerName.
+	clientForwardAddress = clientCommand.Flag("target", "Address to foward connections to (HOST:PORT).").PlaceHolder("ADDR").Required().String()
+	clientUnsafeListen   = clientCommand.Flag("unsafe-listen", "If set, does not limit listen to localhost, 127.0.0.1 or [::1].").Bool()
```

Comment:
`Address and port to listen on (HOST:PORT), or unix:PATH.` (in client mode, unix sockets are allowed for the --listen param, in server mode it's for the --target parameter).